index.html

```diff
@@ -565,6 +565,7 @@
 The time immediately after the user agent receives the last byte of the response or immediately before the transport connection is closed, whichever comes first. The resource here can be received either from <a
 href="https://www.w3.org/TR/html5/browsers.html#relevant-application-cache">relevant application caches</a>, local resources, or from the server.
 </li>
+<li>The time immediately before the user agent aborts the fetch due to a response failure.</li>

```

Comment:
I think we can use "due to a [network error](https://fetch.spec.whatwg.org/#concept-network-error)". 

However, now I'm wondering what the "immediately before transport connection closed above" is covering, because that also seems like a network error =/